docs/README.md

```diff
@@ -69,6 +69,7 @@ The User Guide describes how to configure Triton, organize and configure your mo
 * Collecting Server Metrics [[Overview](README.md#metrics) || [Details](user_guide/metrics.md)]
 * Supporting Custom Ops/layers [[Overview](README.md#framework-custom-operations) || [Details](user_guide/custom_operations.md)]
 * Using the Client API [[Overview](README.md#client-libraries-and-examples) || [Details](https://github.com/triton-inference-server/client)]
+* Cancelling Inflight Inference Request [[Overview](README.md#cancelling-inflight-inference-request) || [Details](user_guide/request_cancellation.md)]
```

Comment:
```suggestion
* Cancelling Inference Requests [[Overview](README.md#cancelling-inference-requests) || [Details](user_guide/request_cancellation.md)]
```

Suggest to avoid using the term "inflight" if we can avoid it -as that will be used again with inflight batching and may be confusing. 

Does "Cancelling Inference Requests" work in its place? Is clear enough?

Reply:
I guess so.